src/StateMachine/Resolver.php

```diff
@@ -1,160 +0,0 @@
-<?php
```

Comment:
Two questions:
1) Can you explain why this could be removed?
2) What happens if I complete a payment in Sylius now? Will it try to capture the payment?

Reply:
I removed it because besides sending the capture and refund request it was doing the same thing [OrderPaymentStateResolver](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Component/Core/StateResolver/OrderPaymentStateResolver.php ) does.
Right now completing a payment in Sylius will not send a capture request.

Reply:
As you can see in the resolver we had this `disableCapture` thing. Where/when are payments captured in Sylius?